account_banking_mandate/__manifest__.py

```diff
@@ -1,13 +1,13 @@
-# Copyright 2014 Compassion CH - Cyril Sester <csester@compassion.ch>
-# Copyright 2014 Tecnativa - Pedro M. Baeza
-# Copyright 2015-16 Akretion - Alexis de Lattre <alexis.delattre@akretion.com>
-# Copyright 2017 Tecnativa - Carlos Dauden <carlos.dauden@tecnativa.com>
+# Copyright 2014-2020 Compassion CH - Cyril Sester <csester@compassion.ch>
+# Copyright 2014-2020 Tecnativa - Pedro M. Baeza
+# Copyright 2015-2020 Akretion - Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2017-2020 Tecnativa - Carlos Dauden <carlos.dauden@tecnativa.com>
```

Comment:
You do you change the Copyright from the previous changes? This should be left as is, and you should only add your Copyright with the current year